### DISCUSSION TITLE
表格识别两个单元格文字靠得很近识别成一个单元格

### DISCUSSION THREAD

**Discussion by hf62580** [PaddlePaddle/PaddleOCR#12348] [Q&A]
表格识别两个单元格文字靠得很近识别成一个单元格
![QQ截图20240227183113](https://github.com/PaddlePaddle/PaddleOCR/assets/52707280/c563e984-a968-4d00-bf51-7ebbdc6b411f)
比如第二行会识别成
<td><td>050020996923174000253<td>美诺速冻薯条800g<td>袋<td>12|<td>1<td>12. 000<td>15. 00000<td><td><td><td>
这个有没有什么改善的方法

**Comment by hf62580** [comment]
050020996923174000253合并成一个单元格了，而不是分开两个

**Comment by tink2123** [comment]
感谢关注，目前表格识别模型的精度不够准确，我们还在优化可以期待下一个版本的模型效果。

**Comment by zhuxiaobin** [comment]
@tink2123 其他模型都不错，现有所有模型中，表格识别模型准确率精度的确达不到，下面这张图中混合多种情况，现有模型没办法完美识别和还原
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/5206194/230dcfb5-0ac3-4ec2-93d3-fc9cf329f506)

**Comment by zhangqiqi1228** [comment]
> 感谢关注，目前表格识别模型的精度不够准确，我们还在优化可以期待下一个版本的模型效果。

@tink2123 请问下一个版本大概什么时候会更新呢？
以及对于表格识别有什么优化的思路和建议么？可以推荐一些表格识别效果较好的开源模型么？谢谢

**Comment by tink2123** [comment]
下一个版本预计会在4月中旬发布，目前中文开源表格模型，可以试一下TableMaster。 另外大模型领域也都在做表格识别，感兴趣也可以关注一下

**Comment by jiayan2024** [comment]
@tink2123 可以推荐一些表格识别效果较好的英文开源模型么？

**Comment by hf62580** [comment]
> 下一个版本预计会在4月中旬发布，目前中文开源表格模型，可以试一下TableMaster。 另外大模型领域也都在做表格识别，感兴趣也可以关注一下

请问下一个版本什么时候发布
